samples/contrib/aws-samples/hosting_model_monitor_pipeline/README.md

```diff
@@ -0,0 +1,99 @@
+# Host a Model and Create a SageMaker Model Monitor
+
+This sample demonstrates a Kubeflow pipeline that
+- host a machine learning model in Amazon SageMaker
+- monitor a live endpoint for violations against constraints
+
+## Prerequisites
+Follow the steps in [Sample AWS SageMaker Kubeflow Pipelines](../README.md#inputs-to-the-pipeline) 
+
+### Create an IAM Role
+Follow [SageMaker execution role](../README.md#inputs-to-the-pipeline) and create an IAM role for SageMaker execution.
+
+### Create an S3 Bucket
+To setup an endpoint and create a monitoring schedule, we need an S3 bucket to store model and baseline data. Run the following commands to create an S3 bucket. Specify the value for `SAGEMAKER_REGION` as the region you want to create your SageMaker resources. For ease of use in the samples (using the default values of the pipeline), we suggest using `us-east-1` as the region.
+
+```
+export SAGEMAKER_REGION=us-east-1
+export S3_BUCKET_NAME="data-bucket-${SAGEMAKER_REGION}-$RANDOM"
```

Comment:
RANDOM's range is between 0-32767, can you change the prefix to "kfp-sm-data-bucket" to reduce the chance of collisions.